web/src/components/ThemeContext.tsx

```diff
@@ -8,7 +8,7 @@ import buildConfig from '../constants/buildConfig'
 type ThemeType = 'light' | 'contrast'
 
 export type ThemeContextType = {
-  theme: ReturnType<typeof buildConfig>['lightTheme'] | ReturnType<typeof buildConfig>['highContrastTheme']
+  theme: ReturnType<typeof buildConfig>['lightTheme'] | ReturnType<typeof buildConfig>['contrastTheme']
```

Comment:
🔧 The correct type here would be `ThemeType`. However, in order to use it, you need to rename `ThemeType` above in L8 to e.g. `ThemeName` or `ThemeKey` and import `ThemeType` from `build-configs`.

Reply:
Good idea! I placed it in an extra file in build-configs...